packages/alchemy/src/middleware/gas-manager.ts

```diff
@@ -21,27 +28,54 @@ export interface AlchemyGasManagerConfig {
  *
  * @param provider - the smart account provider to override to use the alchemy gas manager
  * @param config - the alchemy gas manager configuration
- * @param estimateGas - if true, this will use `alchemy_requestGasAndPaymasterAndData` else will use `alchemy_requestPaymasterAndData`
+ * @param deleteGasEstimation - if true, this will use `alchemy_requestGasAndPaymasterAndData` else will use `alchemy_requestPaymasterAndData`
  * @returns the provider augmented to use the alchemy gas manager
  */
 export const withAlchemyGasManager = (
   provider: AlchemyProvider,
   config: AlchemyGasManagerConfig,
-  estimateGas: boolean = true
+  deleteGasEstimation: boolean
```

Comment:
nit: can we call this something like `withGasEstimation`? delete gas estimation is odd

Reply:
oh it's called `delegateGasEstimation` further down; let's call it that here too